ui/src/components/yaml/index.tsx

```diff
@@ -45,6 +45,7 @@ const Yaml = (props: IProps) => {
   const yamlRef = useRef<LegacyRef<HTMLDivElement> | undefined>()
   const diagnosisContentRef = useRef<HTMLDivElement>(null)
   const interpretEndRef = useRef<HTMLDivElement>(null)
+  const contentRef = useRef<HTMLDivElement>(null)
```

Comment:
Could you clarify the purpose of adding this new ref `contentRef`? It would be helpful to understand how it will be used in the component.